uawidgets/new_node_dialogs.py

```diff
@@ -69,12 +78,13 @@ def get_nodeid_and_bname(self):
         if self.nodeidCheckBox.isChecked():
             nodeid = ua.NodeId(namespaceidx=ns)
         else:
-            nodeid = ua.NodeId.from_string(self.nodeidLineEdit.text())
+            ns_nodeid = "ns={};".format(ns) + self.nodeidLineEdit.text()
+            nodeid = ua.NodeId.from_string(ns_nodeid)
```

Comment:
No, we cannot do that. we can use the namespace widget to set default namespace, but we need to let user set different namespaces for bname and nodeid. if they know the string syntax they can write ns=xx